### ISSUE TITLE
[RuntimeException]   Missing $settings['hash_salt'] in settings.php.

### ISSUE THREAD

**Issue by jmolivas** [hechoendrupal/DrupalAppConsole#293]
When running drupal-console from a drupal directory but drupal is not installed yet, this error is shown:
```
  [RuntimeException]
  Missing $settings['hash_salt'] in settings.php.
```

**Comment by jmolivas** [comment]
Planned solution add a error message:
```
  [Exception]
  Could not find a Drupal settings.php file at sites/default/settings.php.
```
This is how look before and after:
![drupal-missing-settings](https://cloud.githubusercontent.com/assets/366275/5685732/1c764e2c-97f2-11e4-837f-bbdb4298880e.png)

**Comment by jmolivas** [comment]
Update console to show commands before the error message

![drupal-missing-settings-final](https://cloud.githubusercontent.com/assets/366275/5693045/4edb7bce-98c1-11e4-9b78-414503bc46c0.png)

**Comment by jmolivas** [comment]
Fixed on #287
Message
 ```
[Exception]
  In order to list all of the available commands, you should run this inside
  a drupal root directory.
```
![drupal-outsideroot-final](https://cloud.githubusercontent.com/assets/366275/5729845/73a980e8-9b2a-11e4-8f23-f52c883d07c3.png)

The message may be changes form `Exception` to `Notice` or something more user friendly later

**Comment by jmolivas** [comment]
Fixed on #287
Message
 ```
[Exception]
  In order to list all of the available commands you should install drupal first.
```
![screen shot 2015-01-13 at 1 49 45 pm](https://cloud.githubusercontent.com/assets/366275/5729935/1d63d174-9b2b-11e4-8f55-f526d5146dc7.png)
The message may be changes form `Exception` to `Notice` or something more user friendly later
